app/src/main/res/layout/modal_choose_delivery_address.xml

```diff
@@ -0,0 +1,192 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_cancel_black_24dp"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="3dp"/>
+
+    <View
+        android:id="@+id/packageInfoTitleDivider"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_below="@id/packageInfoTitle"
+        android:layout_marginBottom="16dp"
+        android:background="@color/colorDivider" />
+
+    <EditText
+        android:id="@+id/pickupAddressEditText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="Type here the pickup address"
+        android:layout_below="@id/packageInfoTitleDivider"
+        android:background="@color/transparentWhite10"
+        android:gravity="center"
+        android:textColor="@color/colorDivider"
+        android:textColorHint="@color/colorDivider"
+        android:textSize="16dp"
+        android:padding="14dp"
+        android:layout_marginBottom="16dp"/>
+
+    <RelativeLayout
+        android:id="@+id/commonAddressesContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/pickupAddressEditText"
+        android:background="@color/transparentWhite05"
+        android:padding="8dp"
+        android:layout_marginBottom="16dp">
+
+        <TextView
+            android:id="@+id/receiverHeaderEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:text="COMMON ADDRESSES"
+            android:textStyle="bold"
+            android:textColor="@color/colorTextIcons"
+            android:padding="8dp"
+            android:layout_marginBottom="8dp"/>
+
+        <LinearLayout
+            android:id="@+id/commonAddressesList"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/receiverHeaderEditText"
+            android:orientation="vertical"
+            android:layout_marginBottom="0dp">
+
+
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Rua Sen. Fraga 1001 - Manguinhos"
+                android:textColor="@color/colorDivider"
+                android:textColorHint="@color/colorDivider"
+                android:background="@color/transparentWhite05"
+
+                android:textSize="16dp"
+                android:padding="8dp"
+                android:layout_marginBottom="8dp"/>
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:hint="Av. Gomes Freire 290 - Centro"
```

Comment:
This should be in the String file with the name: modalDeliveryAddress_txtView_hintCommonAddresses2.